### DISCUSSION TITLE
Quectel and SIMCom modem power

### DISCUSSION THREAD

**Discussion by improperaffo** [srsran/srsRAN_Project#606] [General Help] [answered]
Dear all,
We have encountered some issues with our discrete modems. We own different models including the Quectel RM500Q-GL, the Quectel RM502Q-AE, and the SIMCom SIM8200EA-M2.

They all connect reliably to our 5G SA network, though, if we try generating some mildly intensive traffic (e.g., ping with an interval of 2 ms, running a speedtest, or iperf3) the modems seem to reboot instantly. 
We have connected the same modem to some commercial 4G networks and we did not observe such behaviour. 

We read online that it might be a power issue, yet we have tried using an external power supply (20 W) and that did not solve our issue. 

If anybody else is using those modems, are they also experiencing the same issues?

Thanks,
Raphael

**Comment by ismagom** [comment]
Hi,

Are you using the latest software release?

**Reply by improperaffo** [reply]
If you mean SRS, yes, we are using the latest tag (24.04)

**Comment by improperaffo** [comment, marked as answer]
Somehow the SDR was the cause of this problem, we are not entirely sure what caused it, but switching to a USRP b210 solved all the issues.
